src/main/java/racing/Car.java

```diff
@@ -0,0 +1,25 @@
+package racing;
+
+public class Car {
+
+    private static final int LOWER_BOUND = 4;
+    private static final int UPPER_BOUND = 9;
+    private Position position;
```

Comment:
```suggestion
    private static final int UPPER_BOUND = 9;
    
    private Position position;
```

클래스 변수와 인스턴스 변수 사이에 공백을 추가한다면 가독성이 상승합니다 :)